plugin/src/main/groovy/com/btkelly/gnag/models/Violation.java

```diff
@@ -89,6 +89,10 @@ public String getRelativeFilePath() {
     public Integer getFileLineNumber() {
         return fileLineNumber;
     }
+    
+    public boolean hasAllLocationInfo() {
+        return getRelativeFilePath() != null && getFileLineNumber() != null;
```

Comment:
If this returns false, this comment necessarily forms part of the aggregated comment.